lldb/source/API/SBSymbol.cpp

```diff
@@ -153,6 +153,14 @@ lldb_private::Symbol *SBSymbol::get() { return m_opaque_ptr; }
 
 void SBSymbol::reset(lldb_private::Symbol *symbol) { m_opaque_ptr = symbol; }
 
+SBAddress SBSymbol::GetRawAddress() {
+  LLDB_RECORD_METHOD_NO_ARGS(lldb::SBAddress, SBSymbol, GetRawAddress);
+
+  SBAddress addr;
+  addr.SetAddress(&m_opaque_ptr->GetAddressRef());
```

Comment:
Could you have used `m_opaque_ptr->GetIntegerValue` to directly fetch the integer value?

Reply:
Indeed, and I renamed GetRawAddress into GetIntegerValue.